include/psen_scan_v2/scanner_configuration.h

```diff
@@ -51,12 +52,12 @@ class ScannerConfiguration
 
 private:
   boost::optional<uint32_t> host_ip_;
-  boost::optional<uint16_t> host_data_port_;
-  boost::optional<uint16_t> host_control_port_;
+  uint16_t host_data_port_{ constants::DATA_PORT_OF_HOST_DEVICE };
```

Comment:
Maybe introduce an alias for the port type?

Reply:
I think we can discuss this independently from this pull request.